lib/config/index.spec.ts

```diff
@@ -97,6 +97,13 @@ describe('config/index', () => {
       ]);
     });
 
+    it('falls back from yarn to pnpm', () => {
```

Comment:
Do you really mean to have this PR title?

`feat(5607)!: separate npm, pnpm, yarn managers`

Is `(5607)` really the scope you want to use?